tests/src/main.F90

```diff
@@ -895,7 +896,7 @@ program main
   case("prg_system_parse_write_xyz")
      call prg_parse_system(mol,"coords_100","xyz")
      call prg_write_system(mol, "mysystem","xyz")
-     call system("diff -qs  mysystem.xyz coords_100.xyz > tmp.tmp")
+     call system("diff -qs --ignore-space-change mysystem.xyz coords_100.xyz > tmp.tmp")
```

Comment:
Alternatively, you could use `--ignore-all-space` which also includes changes in trailing whitespace.

Reply:
I'll make that change, thanks!